app/components/developer_primary_action_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class DeveloperPrimaryActionComponent < ApplicationComponent
+  def initialize(user:, developer:)
+    @user = user
+    @developer = developer
+  end
+
+  def owner?
+    @user&.developer == @developer && !@developer.nil?
```

Comment:
```suggestion
    @user&.developer == @developer && @developer.present?
```

Reply:
Does this also need a `#persisted?` check like `#business?`?

Reply:
Ah, that makes sense. I'm find without the `#persisted?` check then. Let's apply the `nil` vs. `present?` and wrap this up.

Reply:
I think we can entirely eliminate it for the same reason as `persisted?`. You can check our my recent commit to see what the could look like. Either way I think we're fine. If you don't care one way or another, I'll make your suggested change in the interest of wrapping this up.